test/e2e.sh

```diff
@@ -39,10 +39,10 @@ runWebserver 8090
 WS_EXT_PID=$PID
 
 echo ""
-echo "Fiddle App"
-cd $BASE_DIR/../website/fiddle
-killWebserver 8080
-runWebserver 8080 public /bundle.js
+echo "Js Test App"
```

Comment:
In addition if we are not using Fiddle anymore for the e2e tests than we might also have to change something in the **testCompatibility.sh** file. 
I remember vaguely that @viktorsperling was working on something there [fixing the broken compatibility tests](https://github.com/SAP/luigi/pull/2780/commits/b0d66673744fb02d948c98fe027242d469d86650). It then seems like we might need to add the new 4500 js-app in [there](https://github.com/SAP/luigi/blob/master/scripts/testCompatibility.sh#L58) as well so that the compatibility tests keep working.

Reply:
First we need to release this branch and after that we can adjust the script for the compatibility tests. 
Created a [followup ticket](#2872) for this.